apps/participants/views.py

```diff
@@ -386,3 +395,33 @@ def remove_self_from_participant_team(request, participant_team_pk):
         if participants.count() == 0:
             participant_team.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@api_view(["GET"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def participant_team_detail_in_challenge(request, challenge_pk):
```

Comment:
Please update the view function name accordingly to `get_participant_team_details_for_challenge`.